.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+**Note: for support questions, please use Discord**. This repository's issues are reserved for feature requests and bug reports.
+
+* **I'm submitting a ...**
+  - [ ] bug report
+  - [ ] feature request
+  - [ ] support request => Please do not submit support request here, see note at the top of this template.
+
+* **Do you want to request a *feature* or report a *bug*?**
+
+* **What is the current behavior?**
+
+* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via
+https://imgur.com/ or similar.
+
+* **What is the expected behavior?**
+
+* **What is the motivation / use case for changing the behavior?**
+
+* **Please tell us about your environment:**
+  
+  - Prism Version: X.X.X
+  - Server Type: [Spigot|Paper|CraftBukkit]
+  - Server Version: X.X.X
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5470889)